R/summary_ang_dev_circular.R

```diff
@@ -0,0 +1,31 @@
+#' Angular Deviation
+#'
+#' @description  This function calculates the angular deviation of a circular data vector.
+#'
+#' @param x A vector of circular data.
+#' @param na.rm Logical indicating whether to remove missing values (default is FALSE).
+#' @param na_type Character specifying the type of missing values to consider if na.rm is TRUE.
+#' @param ... Additional arguments to be passed to circular::angular.deviation() function.
+#'
+#' @return The angular deviation of the input data.
+#'
+#' @export
+#'
+#' @examples
+#' # Example usage:
+#' data <- circular::circular(c(0, pi/2, pi, 3*pi/2, 2*pi))
+#' summary_ang_dev_circular(data)
+#' 
+#'
+#'# summary_ang_dev_circular(data, na.rm = TRUE, na_type = "na.fail")
+#' 
+#'
+#'# summary_ang_dev_circular(data, na.rm = TRUE, na_type = "na.exclude")
```

Comment:
They couldn't run without the 'na_check' function which is also a stand-alone function within the summary functions. I was hoping that it will work once the package is created. Not sure if that will be the case though.

Reply:
Good point! Thanks